### PR TITLE
fix emoji sizing for context menu, mini context menu and reactions 

### PR DESCRIPTION
**Pull request by daraksha-dk**
### Details

Updated the size of emoji buttons at 3 locations (context menu, mini context menu and reactions)
Also removed the use of "*sizeScale" from the code.

https://github.com/Expensify/App/issues/15815#issuecomment-1472386824
https://github.com/Expensify/App/issues/15815#issuecomment-1484968670
https://github.com/Expensify/App/issues/15815#issuecomment-1490262629
https://github.com/Expensify/App/issues/15815#issuecomment-1490639510
https://github.com/Expensify/App/issues/15815#issuecomment-1491694251

### Fixed Issues

$ https://github.com/Expensify/App/issues/15815
PROPOSAL: https://github.com/Expensify/App/issues/15815#issuecomment-1472386824

### Tests

1. Go to any chat
2. Add a reaction to any message
3. Verify that the size of the emoji is 28px with a font size of 15px
4. Hover over the message to get the mini context menu
5. Verify that the size of the emoji is 20px with a font size of 13px
6. Right click on the message to get the context menu
7. Verify that the size of the emoji is 34px with a font size of 17px

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Go to any chat
2. Add a reaction to any message
3. Verify that the size of the emoji is 28px with a font size of 15px
4. Hover over the message to get the mini context menu
5. Verify that the size of the emoji is 20px with a font size of 13px
6. Right click on the message to get the context menu
7. Verify that the size of the emoji is 34px with a font size of 17px

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

image

image

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/71884990/230944690-ca3eb57d-327d-4da1-adb6-b5f452599f7a.mp4

 

</details>

<details>
<summary>Mobile Web - Safari</summary>

 

https://user-images.githubusercontent.com/71884990/230944074-93579f33-69c3-42a0-98f0-e97d9db28374.mp4

</details>

<details>
<summary>Desktop</summary>

 

https://user-images.githubusercontent.com/71884990/230945096-8f47c957-6301-4777-a388-83da8e3efaa2.mp4

</details>

<details>
<summary>iOS</summary>

image

image
 

</details>

<details>
<summary>Android</summary>

 
image

image

</details>

### CODE REVIEW

File: src/styles/StyleUtils.js [hunk 1]
```diff
@@ -874,42 +874,52 @@ function getEmojiSuggestionContainerStyle(
...
     };
+}
 
+function getEmojiReactionCounterTextStyle(hasUserReacted) {
     if (hasUserReacted) {
-        return {
-            ...sizeStyles,
-            color: themeColors.link,
-        };
+        return {color: themeColors.textLight};
```
Comment (sobitneupane) [inline src/styles/StyleUtils.js:919]: I don't see any proposal or approval regarding this change in color. Can you please add a video to get approval from the design team.

**Review by sobitneupane — COMMENTED** [review]
### Screenshots/Videos
<details>
<summary>Web</summary>

|   |   |
:-------------------------:|:-------------------------:
Screenshot 2023-04-10 at 18 12 45 |  Screenshot 2023-04-10 at 18 12 36
Screenshot 2023-04-10 at 18 12 09 | Screenshot 2023-04-10 at 18 11 47
Screenshot 2023-04-10 at 18 09 34 | Screenshot 2023-04-10 at 18 09 23

https://user-images.githubusercontent.com/25876548/230902048-b90c25e2-eaaf-4bb6-a4c7-ecf2c95e71a3.mov

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/25876548/230905422-528f5120-4bd3-498a-8044-85f50911fec1.mov

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/25876548/230905439-53ded196-2c9f-4d95-a408-b3acc81bc7ee.mov

</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/25876548/230902587-e267ee50-4a13-4f36-94a8-2bcf54f371b4.mov

</details>

<details>
<summary>iOS</summary>

https://user-images.githubusercontent.com/25876548/230905408-006d5df4-0bd1-47e7-b155-bb5b4a9c3ba3.mov

</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/25876548/230905387-a342c44c-3d8b-48cd-b12f-e1eb3b42709a.mov

</details>

**Comment by sobitneupane** [comment]
@shawnborton 
I feel like emojis with 13px look quite smaller in comparison to other icons in context menu.
13px (implemented in this PR):
Screenshot 2023-04-10 at 18 49 04
15px:
Screenshot 2023-04-10 at 18 52 13

And emojis with 15px look quite larger for reaction bubble for mobile devices (not a strong opinion).
| 13px | 15px (implemented in this PR)|
:--:|:--:
Screenshot 2023-04-10 at 18 54 57 | Screenshot 2023-04-10 at 18 56 02
Screenshot 2023-04-10 at 18 54 49 | Screenshot 2023-04-10 at 18 56 08
Web Screenshot 2023-04-10 at 19 00 47 | Web Screenshot 2023-04-10 at 19 00 16

**Comment by shawnborton** [comment]
I agree that in the mini context menu, 15px looks better. 

But for the reaction buttons in the chat, I think 15px looks nice.

**Comment by sobitneupane** [comment]
> I agree that in the mini context menu, 15px looks better.
> 
> But for the reaction buttons in the chat, I think 15px looks nice.

@daraksha-dk Let's change the font size of mini context menu to 15px.

File: src/styles/StyleUtils.js (hunk 1 above)
Comment (sobitneupane) [reply src/styles/StyleUtils.js]: @daraksha-dk There is also change in `backgroundColor` of EmojiReactionBubble on hover which is not present in staging.

https://user-images.githubusercontent.com/25876548/230912707-f5a335a6-e26e-4ce0-9559-5c63b6e4233a.mov

**Comment by sobitneupane** [comment]
@daraksha-dk Thanks for the changes. Will you please update the screenshots to reflect the latest changes?

**Review by sobitneupane — COMMENTED** [review]
### Screenshots/Videos
<details>
<summary>Web</summary>


https://user-images.githubusercontent.com/25876548/230938586-1d5d2f82-bac4-4df4-a75b-416e55394bde.mov


</details>

<details>
<summary>Mobile Web - Chrome</summary>


https://user-images.githubusercontent.com/25876548/230939264-be03c235-cc6a-4c19-9a74-25c6dc2d34ec.mov


</details>

<details>
<summary>Mobile Web - Safari</summary>


https://user-images.githubusercontent.com/25876548/230939229-2870885c-8acd-46ee-b2cc-ad3ac7a5e141.mov


</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/25876548/230938610-c75e429a-2dc1-4d95-9c76-8fcdc8359968.mov



</details>

<details>
<summary>iOS</summary>


https://user-images.githubusercontent.com/25876548/230939244-08a79498-4675-413f-b7bb-64c11f040e17.mov


</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/25876548/230940446-d9b73efe-22d8-456b-923e-eacdc71e1111.mov



</details>

**Review by sobitneupane — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

**Review by stitesExpensify — APPROVED** [review]

**Pull request merged**